lib/config/presets/internal/regex-managers.ts

```diff
@@ -14,4 +14,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  githubActionsVersions: {
+    description:
+      'Update `_VERSION` environment variables in GitHub Action files.',
+    regexManagers: [
+      {
+        fileMatch: ['^.github/workflows/[^\\.]+\\.ya?ml$'],
```

Comment:
Could this be simplified?

```suggestion
        fileMatch: ['^.github/workflows/\\.+\\.ya?ml$'],
```

Reply:
`\\.+\\.ya?ml` would mean 1-n `.` characters, another `.` characters and then `yaml` or `yml`, or am I misreading this? 😅

Reply:
Yeah, I misread this regex. My bad!

Currently it matches any number of characters that is not `.`, followed by `.yaml` or `.yml`? What about `\S+` instead?

Reply:
`\S+` allows for slashes and `.` characters. I've changed it now to allow `.` characters, but prevent `/`, since I don't think GitHub supports workflows in subfolders.